net/http/src/THttpServer.cxx

```diff
@@ -32,8 +32,7 @@
 
 #include <string>
 #include <cstdlib>
-#include <stdlib.h>
-#include <string.h>
+#include <cstring>
```

Comment:
Might need prefix of `std::` for `strstr` etc.

Reply:
It is compiled on all platforms. Once it is really appears - I will fix it.